examples/src/main/java/io/grpc/examples/errorhandling/ErrorHandlingClient.java

```diff
@@ -67,11 +67,11 @@ public static void main(String [] args) throws Exception {
     new ErrorHandlingClient().run();
   }
 
-  private Server server;
   private ManagedChannel channel;
 
   void run() throws Exception {
-    server = ServerBuilder.forPort(0).addService(new GreeterGrpc.GreeterImplBase() {
+    // launch a server on a port number that is available.
```

Comment:
How about:

// Port 0 means that the operating system will pick an available port to use.